CRM/Contact/Form/Task/Email.php

```diff
@@ -131,11 +131,7 @@ public function preProcess() {
     if (!$cid && $this->_context != 'standalone') {
       parent::preProcess();
     }
-
-    //early prevent, CRM-6209
-    if (count($this->_contactIds) > CRM_Contact_Form_Task_EmailCommon::MAX_EMAILS_KILL_SWITCH) {
-      CRM_Core_Error::statusBounce(ts('Please do not use this task to send a lot of emails (greater than %1). We recommend using CiviMail instead.', array(1 => CRM_Contact_Form_Task_EmailCommon::MAX_EMAILS_KILL_SWITCH)));
-    }
+    CRM_Contact_Form_Task_EmailCommon::bounceIfSimpleMailLimitExceeded(count($this->_contactIds));
```

Comment:
I liked what Andrew  suggested about using quick instead of simple

Reply:
just found this PR via [SE question](https://civicrm.stackexchange.com/questions/21073/how-to-change-hard-coded-limit-of-50-transactional-emails)

+1 eileens' comment and also consider: if a site starts trying to send 1000 emails in a single request, this will not be "quick mail" at all (it'll probably start being "timeouty mail")